pegasus/sites.v3/code.org/public/hourofcode2022.haml

```diff
@@ -0,0 +1,103 @@
+---
+title: Explore, Play, and Create with Computer Science
+theme: responsive
+---
+
+%link{href: "/css/generated/hoc2022-landing-page.css", rel: "stylesheet"}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
+
+-# FontAwesome icons
+- icon_rocket = "fa-regular fa-rocket"
+- icon_futbol = "fa-regular fa-futbol"
+- icon_paint = "fa-regular fa-paintbrush-pencil"
+
+%header
+  .wrapper
+    %span=I18n.t(:og_title_here)
+    %h1=I18n.t(:hoc2022_banner_title)
+    %h2=I18n.t(:try_it_yourself)
+    %nav
+      %ul
+        %li
+          %i{class: "#{icon_rocket}"}
+          %a{href: "#explore"}
+            =I18n.t(:hoc2022_landing_page_explore_title)
+        %li
+          %i{class: "#{icon_futbol}"}
+          %a{href: "#play"}
+            =I18n.t(:hoc2022_landing_page_play_title)
+        %li
+          %i{class: "#{icon_paint}"}
+          %a{href: "#create"}
+            =I18n.t(:hoc2022_landing_page_create_title)
+  %img.left{src: "/images/hoc2022-landing-page-banner-left.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_left)}
+  %img.right{src: "/images/hoc2022-landing-page-banner-right.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_right)}
+
+%main
+  -# Hiding the NASA banner until post-launch due to mktg reasons
+  -# %aside
+    %img{src: "/images/nasa-logo.svg", alt: "NASA logo"}
+    %h1
+      Copy from NASA goes here
```

Comment:
Since the %aside is commented out, does that mean none of this shows?

Thoughts about keeping this code in a separate branch until it's launched? Unless you're planning to launch it very soon

Reply:
Correct, we're planning to launch this about a week after next week's launch once we get more info from NASA — and if we end up not using it all I can delete it.

Reply:
Okay –– sounds like a plan! 👍